lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -236,6 +239,51 @@ def test_dataset_list(self, server_config, query_as, login, query, results):
         result = query_as(query, login, HTTPStatus.OK)
         self.compare_results(result.json, results, query, server_config)
 
+    @pytest.mark.parametrize(
+        "login,query,message",
+        (
+            (None, {"mine": True}, "'mine' filter requires authentication"),
+            (None, {"mine": "no"}, "'mine' filter requires authentication"),
+            (
+                "drb",
+                {"mine": "yes", "owner": "drb"},
+                "'owner' and 'mine' filters cannot be used together",
+            ),
+            (
+                "drb",
+                {"mine": False, "owner": "test"},
+                "'owner' and 'mine' filters cannot be used together",
```

Comment:
I think we need a test for `access` and `mine` combination as well.

Reply:
Good catch. `?mine&access=` doesn't diverge functionally from the existing `?owner=<me>&access=`, but it wouldn't hurt to (redundantly) test that logic through the new path. However, `?mine=false&access=` isn't the same as `?access=` alone so, yeah; I need a test case there.